packages/connect-explorer/src/middlewares/trezorConnectMiddleware.ts

```diff
@@ -12,7 +12,10 @@ export const trezorConnectMiddleware =
         next(action);
 
         if (action.type === ON_LOCATION_CHANGE && !prevConnectOptions) {
+            // TODO: just for dev! still need to figure out best way to pass proper src to web-extension version of connect-explorer
             const connectSrc = getQueryVariable('src');
+            // const connectSrc = 'http://localhost:8088';
+            console.log('connectSrc in connect-explorer middleware', connectSrc);
```

Comment:
This is just for now.
I think it might be a good idea to add a new option in the settings page of connect-explorer to setup custom connectSrc that can be used for testing.

Reply:
yes. TrezorConnect.init method form should be somehow incorporated into connect-explorer anyway. It should start with some interactive example, 
1. dear developer, please first do TrezorConnect.init and chose from available options
2. now you may call any of the api methods you like

Reply:
I added connectSrc to settings page in connect-explorer that can be very handy for tests.